es-app/src/CollectionSystemManager.cpp

```diff
@@ -199,20 +255,17 @@ void CollectionSystemManager::updateSystemsList()
 		SystemData::sSystemVector.push_back(mCustomCollectionsBundle);
 	}
 
```

Comment:
Just to confirm, there is a functional change here that I'm not sure is deliberate.

Until now, the automatic collections were always at the end of the system list, acting as quick indexes to the library. The order was always:
1 - Regular Systems
2 - Custom Systems
3 - Auto Systems [My Collections, All, Favorites, Last Played]

If they were sorted, the order was
1 - Regular and Custom Systems, sorted
2 - RetroPie
3 - Auto Systems [My Collections, All, Favorites, Last Played]

The way I'm reading this it seems you're dropping this altogether. Am I reading it right?

If it was deliberate, I'd strongly vouch against that, and I'm happy to elaborate and discuss further. That has been the way users have requested it, and from a usability perspective it makes sense that these "index" collections are more easily - and consistently - accessed at he same location rather than mixed in with the remaining systems.

Let me know your thoughts.

Reply:
True, for much of this refactoring I was trying to get rid of the differences between Auto and Custom collections, so this is a logical result of that mind-set, and an intentional change.

However, I was unaware of discussions on the topic of sorting of systems in particular, and if there are strong feelings on this (minor?) topic I will gladly defer to the will of the people.
It will mean I need to think of another implementation of a sort function though.

Reply:
So I've been working on this a bit now, and I have the following sort functions:
```
bool splitSystemSort(SystemData* sys1, SystemData* sys2)
{
	// The goal here is to sort as follows:
	// 1- all regular systems (non-collection), in alphabetical order
	// 2- RetroPie system
	// 3- custom systems and auto systems, in alphabetical order
```
and
``` 
bool mixedSystemSort(SystemData* sys1, SystemData* sys2)
{
	// The goal here is to sort as follows:
	// 1- all regular systems (non-collection) and custom systems, in alphabetical order
	// 2- RetroPie system
	// 3- auto systems, in fixed order: "RETROPIE", "COLLECTIONS", "ALL", "FAVORITE", "RECENT", "KIDGAME", "HIDDEN"

```
Is that about right?